pom.xml

```diff
@@ -291,4 +291,11 @@
 		</profile>
 	</profiles>
 
+	<distributionManagement>
+		<repository>
+			<id>bintray-awspring</id>
+			<url>https://api.bintray.com/maven/awspring/releases/spring-cloud-aws/;publish=1</url>
```

Comment:
format is `https://api.bintray.com/maven/awspring/{{repository}}/{{package}}/;publish=1` in terms of bintray.
/cc @maciejwalkowiak

Reply:
👍